File: JDBCBank/src/main/java/com/revature/JDBCBank/Account.java
package com.revature.JDBCBank;

// Abstract out account operations 
public class Account {

	// Throw custom exception for username already exists
	public void newAccount(String username, String password) throws UserNameExistsException {

	}
	
	// Delete an account
	public void deleteAccount(String username, String password, Integer AccountID) {
		
	}

	// Display accounts
	public String displayAccount(Integer userid) {
		return "";
	}

	// Deposit money to account
	public void deposit(Integer userid, Integer accountid, Double amount) {
		//
	}

	public Double withdraw(Integer userid, Integer accountid, Double amount) {
		return 0.0D;
	}
	

}

File: TsBank/account.ts
export class Account {
    balance : number;
    type : string;
    constructor(balance : number = 0, type : string){
        this.balance = balance;
        this.type = type;
    }

    withdraw(amount : number) {
        this.balance = this.balance - amount;
    }

    deposit(amount : number) {
        this.balance = this.balance + amount;
    }
}